src/Symfony/Component/Lock/Tests/Store/MongoDbStoreTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Tests\Store;
+
+use MongoDB\Client;
+use Symfony\Component\Lock\Store\MongoDbStore;
+
+/**
+ * @author Joe Bennett <joe@assimtech.com>
+ * @requires extension mongodb
+ */
+class MongoDbStoreTest extends AbstractStoreTest
+{
+    use ExpiringStoreTestTrait;
+
+    public static function setupBeforeClass()
+    {
+        $client = self::getMongoConnection();
+        $client->listDatabases();
+    }
+
+    /**
+     * @return Client
+     */
+    private static function getMongoConnection(): Client
+    {
+        return new Client('mongodb://'.getenv('MONGODB_HOST'));
+    }
+
+    /**
+     * @return int
+     */
+    protected function getClockDelay()
+    {
+        return 250000;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStore()
```

Comment:
sure

Reply:
@jderusse since `AbstractStoreTest` doesn't yet specify the return type should we go ahead an upgrade all missing return types from `Symfony\Component\Lock`? What's the policy on doing this at the moment?